docs/reference/fields.txt

```diff
@@ -1304,6 +1304,19 @@ a ``ReadonlyAttribute`` exception will be raised:
    band.update_attribute(:name, "Tool") # Raises the error.
    band.remove_attribute(:name) # Raises the error.
 
+Assignments to the setters of read-only attributes will be ignored:
```

Comment:
Assignments to read-only attributes will be ignored

One doesn't assign to a setter, the setter implements assignment.